endor/src/index.js

```diff
@@ -1,39 +1,57 @@
 import express from 'express';
+import helmet from 'helmet';
 import logger from 'morgan';
 import cookieParser from 'cookie-parser';
 import bodyParser from 'body-parser';
+import session from 'express-session'
+
+import * as auth from './routes/auth.routes';
+import * as client from './routes/client.routes';
 import index from './routes/index.routes';
 import * as projects from './routes/projects.routes';
 import * as contributors from './routes/contributors.routes';
 import * as users from './routes/users.routes';
 import ProjectService from './services/projects.service';
-import sequalize from './db/sequelize';
+import sequelize from './db/sequelize';
 import { getActiveLogger } from './utils/winston';
 import * as owners from './routes/owners.routes';
 import UserService from './services/users.service';
+import AuthService from './services/auth.service';
+import ClientService from './services/client.service';
 
 const app = express();
 
 // middleware //
+app.use(helmet());
 app.use(logger('dev'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
 
+// TODO - reexamine if this is necessary for oauth2orize
+app.use(session({
+  secret: 'Super Secret Session Key',
+  saveUninitialized: true,
+  resave: true
+}));
+
```

Comment:
Did we reexamine? We should do this before merging. If it's done, please remove TODO